examples/web-chat/src/config.ts

```diff
@@ -0,0 +1,13 @@
+import { Protocols } from "@waku/interfaces";
+
+export const CONTENT_TOPIC = "/toy-chat/2/huilong/proto";
+
+const PUBLIC_KEY = "AOGECG2SPND25EEFMAJ5WF3KSGJNSGV356DSTL2YVLLZWIV6SAYBM";
+const FQDN = "test.waku.nodes.status.im";
+export const ENR_TREE = `enrtree://${PUBLIC_KEY}@${FQDN}`;
+
+export const PROTOCOLS = [
```

Comment:
This is not needed.

Reply:
I think for the example it is good to make this part as explicit as possible.

Reply:
I don't, we want to provide good default so that the library consumer does not have to "learn one more thing" when it's actually not needed. Making the learning curve as flat as possible.

Reply:
This is a good point, I agree.